infra/ibmcloud/terraform/k8s-s390x-infra-setup/README.md

```diff
@@ -0,0 +1,47 @@
+# _TF: IBM K8s Account Infrastructure_
+This Terraform configuration sets up an organized structure for deploying various IBM Cloud resources using modules. 
+
+---
+# To run the automation, follow these steps in order:
+
+**1. Navigate to the correct directory**
+<br> You need to be in the `k8s-s390x-infra-setup` directory to run the automation.
+
+**2. Check the `versions.tf` file**
+<br> Set `secret_key` and `access_key` in `versions.tf` to configure the remote S3 backend (IBM Cloud COS).
+
+**3. Initialize Terraform**
+<br> Execute the following command to initialize Terraform in your project directory. This command will download the necessary provider plugins and prepare the working environment.
+```
+terraform init -reconfigure
+```
+
+**4. Check the `variables.tf` file**
+<br> Open the `variables.tf` file to review all the available variables. This file lists all customizable inputs for your Terraform configuration.
+
+`ibmcloud_api_key` is the only required variable that you must set in order to proceed. You can set this key either by adding it to your `var.tfvars` file or by exporting it as an environment variable.
+
+**Option 1:** Set in `var.tfvars` file
```

Comment:
tfvars or `variables.tf` should not be required here in my opinion - just define the configuration plainly, its not a module